### PR TITLE
Fix copies in the tooltips

### PR DESCRIPTION
**Pull request by warenlg**
Fix https://athenianco.atlassian.net/browse/DEV-233

Signed-off-by: Waren Long <waren@athenian.co>

### CODE REVIEW

File: src/js/components/insights/stages/work-in-progress/createdPrs.jsx [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default {
     factory: (computed) => ({
         meta: {
             title: 'Pull Requests Created',
-            description: 'Pull Request throughput through time.'
+            description: 'Pull request throughput through time.'
```
Comment (eiso) [inline src/js/components/insights/stages/work-in-progress/createdPrs.jsx:50]: Requests

File: src/js/pages/pipeline/Pipeline.jsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ export const pipelineStagesConf = [
         metric: 'lead-time',
         stageName: 'leadtime',
         color: palette.stages.leadtime,
-        hint: 'From the 1st commit of the Pull Requests until the code is being used in production',
+        hint: 'From the 1st commit of the pull requests until the code is being used in production',
```
Comment (eiso) [inline src/js/pages/pipeline/Pipeline.jsx:15]: Can we change this one to:

Average time from the initial commit on a pul request until it is released.

File: src/js/pages/pipeline/Pipeline.jsx [hunk 3]
```diff
@@ -75,7 +75,7 @@ export const pipelineStagesConf = [
         metric: 'merging-time',
         stageName: prStage.MERGE,
         color: palette.stages.merge,
-        hint: 'From the moment the Pull Request is approved until it gets merged',
+        hint: 'From the moment the pull request is approved until it gets merged',
```
Comment (eiso) [inline src/js/pages/pipeline/Pipeline.jsx:78]: until it is merged

File: src/js/pages/pipeline/Pipeline.jsx [hunk 4]
```diff
@@ -100,7 +100,7 @@ export const pipelineStagesConf = [
         metric: 'release-time',
         stageName: prStage.RELEASE,
         color: palette.stages.release,
-        hint: 'From the moment the Pull Request gets merged until it is shipped into production',
+        hint: 'From the moment the pull request gets merged until it is shipped into production',
```
Comment (eiso) [inline src/js/pages/pipeline/Pipeline.jsx:103]: until it is released

File: src/js/components/insights/stages/work-in-progress/createdPrs.jsx (hunk 1 above)
Comment (warenlg) [reply src/js/components/insights/stages/work-in-progress/createdPrs.jsx:50]: You mean "Pull requests throughput"?

Comment (eiso) [reply src/js/components/insights/stages/work-in-progress/createdPrs.jsx:50]: Ignore this one

File: src/js/components/insights/stages/merge/abandonedWork.jsx [hunk 5]
```diff
@@ -69,7 +69,7 @@ export default {
     factory: (computed) => ({
         meta: {
             title: 'Abandoned Work',
-            description: `Evaluate the amount of work and time lost in closed PRs. Show the first ${firstNRepos} closed Pull Requests.`
+            description: `Evaluate the amount of work and time spent in closed pull requests.`
```
Comment (dpordomingo) [inline src/js/components/insights/stages/merge/abandonedWork.jsx:72]: question: do you think that it does not worth letting the user know that the list is limited and not complete?

**Pull request merged**